framework/src/actions/SetupMeshAction.C

```diff
@@ -131,6 +122,16 @@ SetupMeshAction::act()
 {
   // Create the mesh object and tell it to build itself
   _mesh = MooseSharedNamespace::static_pointer_cast<MooseMesh>(_factory.create(_type, "mesh", _moose_object_pars));
+
+#ifdef LIBMESH_ENABLE_AMR
+  unsigned int level = getParam<unsigned int>("uniform_refine");
+
+  // Did they specify extra refinement levels on the command-line?
+  level += _app.getParam<unsigned int>("refinements");
+
+  _mesh->uniformRefineLevel() = level;
+#endif //LIBMESH_ENABLE_AMR
+
   _mesh->init();
```

Comment:
Oh, this explains why displaced tests fail. I am guessing the 4th and 5th failed tests will be fixed.